app/services/bulk_upload/sales/year2022/row_parser.rb

```diff
@@ -467,6 +468,15 @@ def validate_data_protection_answered
     end
   end
 
+  def validate_buyers_organisations
+    organisations_fields = %i[field_44 field_45 field_46 field_47]
+    if organisations_fields.all? { |field| attributes[field.to_s].blank? }
```

Comment:
The error message below suggests this should be `any?` — please could you confirm?

Reply:
I think you read the `blank?` as a `present?` perhaps... we add the error message that one must be selected if none of them are